package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tuyapi",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

Comment:
I would say this is technically a breaking change since `nullPayloadOnJSONError` was removed.

Reply:
Fair.  Personally, I want to break it a lot more!